src/emitter/checkstyle.rs

```diff
@@ -52,8 +52,11 @@ where
             if let DiffLine::Expected(message) = line {
                 write!(
                     writer,
-                    r#"<error line="{}" severity="warning" message="Should be `{}`" />"#,
+                    r#"<error {}="{}" {}="{}" severity="warning" message="Should be `{}`" />"#,
```

Comment:
Embedding all of the attribute names directly in the template literal was resulting in it exceeding the limit, causing the `line formatted, but exceeded maximum width (maximum: 100 (see `max_width` option), found: 108)` error, hence this change to the template